card.js

```diff
@@ -52,10 +52,8 @@ axios.get(url, ).then(function(response) {
             }
         })
 
-        console.log(rowData)
-
         let subsets = []
-        rowData.filter(row => row.values[0].formattedValue == idx).forEach((row, rowIndex) => {
+        rowData.filter(row => row.values[row.values.length-1].formattedValue == idx).forEach((row, rowIndex) => {
```

Comment:
just wondering why did you choose the last index?

Reply:
have a private discussion to clarify this issue. We take the last index of the row => the `Dataloader` name.